src/ROUTES.ts

```diff
@@ -355,7 +355,10 @@ const ROUTES = {
     },
     ROOM_INVITE: {
         route: 'r/:reportID/invite/:role?',
-        getRoute: (reportID: string, role?: string) => `r/${reportID}/invite/${role}` as const,
+        getRoute: (reportID: string, role?: string) => {
+            const route = role ? (`r/${reportID}/invite/${role}` as const) : (`r/${reportID}/invite` as const);
```

Comment:
How about a shorter version?
```suggestion
        getRoute: (reportID: string, role?: string) => `r/${reportID}/invite${role ? `/${role}`: ''}` as const,
```

Reply:
@BrtqKr Could you please help to detail which problem here?

> Fixed role undefined case on room members invite

Reply:
when someone didn't have a proper role inside of a report we ended up with `/undefined` inside of the url. It just seems like someone forgot to put a proper fallback in there

Reply:
@BrtqKr  With previous code, we added a fallback value is empty string. Why shouldn't use ?? operator?

Reply:
Not sure what you meant exactly, but two explanations:
1. 
` getRoute: (reportID: string, role?: string) => `r/${reportID}/invite/${role}` as const,`

In the previous version, skipping the role meant getting undefined, which was getting stringified, which resulted in for example
`r/151422415264937/invite/undefined` in the url. 

2. `??` operator works only as a fallback for null and undefined and in this case we want to treat empty string as falsy as well to remove slash